packages/features/bookings/lib/handleSeats/types.d.ts

```diff
@@ -0,0 +1,80 @@
+import type { Prisma } from "@prisma/client";
+
+import type { AppsStatus } from "@calcom/types/Calendar";
+
+import type { Booking } from "../handleNewBooking";
+
+export type BookingSeat = Prisma.BookingSeatGetPayload<{ include: { booking: true; attendee: true } }> | null;
+
+export type NewSeatedBookingObject = {
+  rescheduleUid: string | undefined;
+  reqBookingUid: string | undefined;
+  eventType: NewBookingEventType;
```

Comment:
Some imports are missing in the files which causes type errors

for example, import for `NewBookingEventType` and `OriginalRescheduledBooking`

Reply:
Thank you! I was trying to figure out why it wasn't working. I don't get why my IDE wasn't throwing an error though.